prometheus_hardware_exporter/config.py

```diff
@@ -37,6 +39,28 @@ def validate_level_choice(cls, level: str) -> str:  # noqa: N805 pylint: disable
             raise ValueError(msg)
         return level
 
+    @validator("disable_collector")
+    def validate_disable_collector_choice(  # pylint: disable=E0213
```

Comment:
Same here why we are using cls and providing two ingore here?

Reply:
Please refer to [pydantic's documentation](https://docs.pydantic.dev/latest/usage/validators/)